src/main/webapp/assets/js/tests/indexScript.testrunner.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
It's super cool that you get to see all of your tests on a HTML page! Out of curiosity, is it possible to view all tests on one page rather than creating separate HTML files for each script file? Just wondering since I saw another HTML file for another test!

Reply:
It's possible, but I would recommend against it in case `<including>` two script files that were meant to operate on separate pages conflict with each other. For example, if two onLoads() are defined, it could cause unexpected errors. Leaving them in separate files also leaves the door open for full frontend testing (as in, testing the HTML elements themselves), as described in [this article](https://www.telerik.com/blogs/journey-client-side-testing-javascript)! :)

Reply:
Gotcha!